vm-setup/roles/v1aX_integration_test/templates/cluster.yaml

```diff
@@ -12,17 +12,23 @@ spec:
     serviceDomain: "cluster.local"
   infrastructureRef:
     apiVersion: infrastructure.cluster.x-k8s.io/{{ CAPI_VERSION }}
+{% if CAPI_VERSION == 'v1alpha2' %}
```

Comment:
doesn't it mean v1alpha1 now would have kind: Metal3Cluster

Reply:
Ah for that we have different templates I suppose

Reply:
yes